.eslintrc.json

```diff
@@ -121,6 +121,22 @@
           }
         ]
       }
+    },
+    {
```

Comment:
This rule is only applied to Next.js core code not to user code. The implementation has to be on Next.js itself and needs to work in the webpack resolving, potentially in the `externals` config of `packages/next/build/webpack-config.ts`

Reply:
Ahh okay, so, I'm on it too

Reply:
@timneutkens @Timer , I pushed some edits using externals to check if next/head is imported inside a custom `_document` file, actually, I have a doubt about the way that I used (it actually works fine but...)

I didn't find a proper way to know the file name that imports an external... 🤔